### PR TITLE
fix: impossible contributing guidelines (#608)

### LINKED ISSUE #608: CONTRIBUTING asks for impossible things
Just looked at the CONTRIBUTING file for the first time in a while and was curious to see that it asks contributors of pull requests to:
1. "Assign yourself so we know you're working on it"
2. "If you're still working on it, add the **under development** label"

Unless I'm much mistaken, neither of these are things that the wider community are permitted to do.

### PR DESCRIPTION
**Pull request by maxim-tschumak**
Fixes #608.

### CODE REVIEW

**Comment by maxim-tschumak** [comment]
@roxspring please have a look

**Comment by roxspring** [comment]
LOL - I was hoping that rights to assign and label things might be opened up but clearly this solution also works!

LGTM.

File: CONTRIBUTING [hunk 1]
```diff
@@ -11,13 +11,12 @@ Just make sure that an issue describing the bug or feature does not already exis
 ## Pull Requests
 Follow these steps to contribute your work to Zally:
 
-1. [Open an issue](https://github.com/zalando-incubator/zally/issues) describing the problem or proposed feature. Assign yourself so we know you're working on it.
+1. [Open an issue](https://github.com/zalando-incubator/zally/issues) describing the problem or proposed feature. Ask maitainers (in the issue thread) to assign the issue to you so we know who is working on what.
```
Comment (netme) [inline CONTRIBUTING]: Nice idea, we can also transform our permanent contributors to external collaborators.

**Comment by netme** [comment]
👍

**Comment by ruiaraujo** [comment]
👍

**Pull request merged**